src/codegen/compile.ml

```diff
@@ -7264,11 +7264,11 @@ and compile_exp (env : E.t) ae exp =
 
     | OtherPrim "rts_callback_table_count", [] ->
```

Comment:
Ha! Maybe we should have called it the callback table! But let's stick with this for now.

Reply:
Do we want to rename the rts_ functions too, or just leave them for now?

Reply:
I'll leave them, this is an external interface.